OpenRA.Mods.Common/Traits/BotModules/SquadManagerBotModule.cs

```diff
@@ -306,35 +306,26 @@ void CreateAttackForce(IBot bot)
 
 		void TryToRushAttack(IBot bot)
 		{
-			var allEnemyBaseBuilder = AIUtils.FindEnemiesByCommonName(Info.ConstructionYardTypes, Player);
-
-			// TODO: This should use common names & ExcludeFromSquads instead of hardcoding TraitInfo checks
-			var ownUnits = activeUnits
-				.Where(unit => unit.IsIdle && unit.Info.HasTraitInfo<AttackBaseInfo>()
-					&& !unit.Info.HasTraitInfo<AircraftInfo>() && !Info.NavalUnitsTypes.Contains(unit.Info.Name) && !unit.Info.HasTraitInfo<HarvesterInfo>()).ToList();
+			// Note that Rush Squad is almost the same as Assuslt Squad at present.
```

Comment:
```suggestion
			// Note that Rush Squad is almost the same as Assault Squad at present.
```

Just a typo.

Reply:
Hi here, I am not going to push this PR because It seems somewhat senseless to do this, we can make the rush tick long enough to avoid this bug.

In addition, I have a probably radical idea on change Assault squad into a hit and run squad and make Rush squad into regular attack squad in `CreateAttackForce`. I am test it on TA for now.

This is the [link](https://gitee.com/CastleJing/TA_Engine/commit/da7a84322428911a70bc796efbb350abf59b031d). It is just a initial attempt.